lib/src/value/color.ts

```diff
@@ -344,6 +311,13 @@ export class SassColor extends Value {
   }
 }
 
+// A helper for converting HWB colors to RGB.
+function toRgb(hue: number, scaledWhiteness: number, scaledBlackness: number) {
```

Comment:
should probably be named `hwbToRgb` rather than just `toRgb` to make it clearer (we might have `hslToRgb`)